README.md

```diff
@@ -1,28 +1,49 @@
-## Requirements
+# Requirements
 - protobuf
 - LZMA
 - 7z
 - lz4
-### Linux
-```
+## Linux
+```bash
 apt install unace unrar zip unzip p7zip-full p7zip-rar sharutils rar uudeview mpack arj cabextract file-roller rename
 apt install liblzma-dev python-pip brotli lz4
 pip install backports.lzma protobuf pycrypto
 ```
-### Mac
+## Arch
+```bash
+pacman -S unace unrar zip unzip p7zip sharutils uudeview arj cabextract file-roller
+pacman -S python python-pip brotli lz4
+pip install backports.lzma protobuf pycrypto
 ```
+### For "rename" and "mpack" you need to manually clone and install the packages
+- rename
+```bash
+git clone https://aur.archlinux.org/rename.git
+cd rename 
+makepkg -Acs
+sudo pacman -U rename-1.3-7-x86_64.pkg.tar.xz
+```
+- mpack
+```bash
+git clone https://aur.archlinux.org/mpack.git
+cd mpack
+makepkg -Acs
```

Comment:
A better option would be,
```bash
for package in mpack rename; do
    git clone https://aur.archlinux.org/"${package}"
    cd "${package}" || continue
    makepkg -si --skippgpcheck
    cd - || break
    rm -rf "${package}"
done```

Reply:
It even installs the package on its own?

Reply:
> It even installs the package on its own?

It does , noice af